GlobalTestHelpers/Extensions/InspectableView.swift

```diff
@@ -140,19 +140,21 @@ extension InspectableView {
         try find(BitwardenTextFieldType.self, containing: title, locale: locale)
     }
 
-    /// Attempts to locate a bitwarden text field with the provided accessibility label.
+    /// Attempts to locate a Bitwarden text field with the provided accessibility label.
     ///
     /// - Parameters:
-    ///   - accessibilityLabel: The accessibility label to use while searching for a button.
+    ///   - type: The type of the text field to locate.
+    ///   - accessibilityLabel: The accessibility label to use while searching for the text field.
     ///   - locale: The locale for text extraction.
-    /// - Returns: A `BitwardenTextFieldType`, if one can be located.
+    /// - Returns: An `InspectableView` of the specified type, if one can be located.
     /// - Throws: Throws an error if a view was unable to be located.
     ///
-    func find(
-        bitwardenTextFieldWithAccessibilityLabel accessibilityLabel: String,
+    func find<T>(
+        textFieldType type: T.Type,
```

Comment:
🎨 Given that `T` could be any inspectable view, should we update the parameter label and docs here to make it generic for any view, rather than just text fields?

Reply:
i had this thought too... i can definitely do that!